### ISSUE TITLE
Drag and drop blocks: Block can't be dragged after long-pressing on an arrow button (Only on iOS)

### ISSUE THREAD

**Issue by fluiddot** [WordPress/gutenberg#41665]
## Description

On iOS and only after long-pressing any of the arrow buttons located in the block toolbar, the block can't be dragged although the chip component is displayed. This only happens one time as it allows dragging the same block after trying again to drag.

**NOTE:** This behavior on Android works as expected.

## Step-by-step reproduction instructions

1. Open a post/page.
2. Add multiple blocks.
3. Select one of the blocks.
4. Long-press over one of the arrow buttons located in the block toolbar (i.e. up or down arrow buttons).
5. Dismiss the "Change block position" modal.
6. Long-press over the block.
7. Observe that the chip component is displayed.
8. Drag the block to a new position.
9. Observe that the chip component doesn't follow the finger position and that the block is not moved when dropping it.

## Expected behaviour

A block can be dragged after long-pressing over its content, independently if previously a long-press was done in a different element.

## Actual behaviour

After long-pressing over any of the arrow buttons, the block can't be dragged.

## Screenshots or screen recording (optional)

https://user-images.githubusercontent.com/14905380/173102988-e0a4e4f3-d8d0-439a-a599-6c8dedef5b74.MP4

## WordPress information
- WordPress version: N/A
- Gutenberg version: N/A
- Are all plugins except Gutenberg deactivated? N/A
- Are you using a default theme (e.g. Twenty Twenty-One)? N/A

## Device information
- Device: iPhone 11
- Operating system: iOS 15.4
- WordPress app version: `pr18857-1c22c47`

**Comment by fluiddot** [comment]
In relation to the triaging of this issue, the percentage of users that use the "Move to top" and "Move to bottom" actions, which are triggered by the long-press gesture, is quite low. The numbers are around 0.09%, so most likely this issue won't really affect a lot of users.

**Comment by tyxla** [comment]
This is no longer relevant after the React Native implementation was removed in #78747. Closing.
